src/Frame/DataBuilderFrame.py

```diff
@@ -106,36 +117,118 @@ def GetPathToFiles(self):
+        super().__init__(root, highlightbackground="pink", highlightthickness=2)
         self.root = root
         self.pathToCSV = pathToCSV
         self.df = None
-        self.readData()
+        self.unchosenColumns = []
+        self.chosenColumns = []
+        self.getAllCols()
+        self.unchoseColVar = tk.StringVar(value=self.unchosenColumns)
+        self.choseColVar = tk.StringVar(value=self.chosenColumns)
+
+        self.unchoseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.unchoseColVar)
+        self.unchoseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+        self.choseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.choseColVar)
+        self.choseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+
+        self.addColButton = tk.Button(self, text="Add Column", command=self.addCol)
+        self.addColButton.pack()
+        self.removeColButton = tk.Button(self, text="Remove Column", command=self.removeCol)
+        self.removeColButton.pack()
+
+    def getAllCols(self):
+        self.df = pd.read_csv(self.pathToCSV)
+        for col in self.df.columns:
+            self.unchosenColumns.append(col)
+        self.unchosenColumns.sort()
+        #print('Debug: ', self.unchosenColumns)
+
+    def addCol(self):
+        try:
+            selected_index = self.unchoseColListbox.curselection()[0]
+        except IndexError:
+            return
+
+        col = self.unchoseColListbox.get(selected_index)
+        self.chosenColumns.append(col)
+        self.unchosenColumns.remove(col)
+        self.updateWindow()
+
+    def removeCol(self):
+        try:
+            selected_index = self.choseColListbox.curselection()[0]
+        except IndexError:
+            return
+
+        col = self.choseColListbox.get(selected_index)
+        self.unchosenColumns.append(col)
+        self.chosenColumns.remove(col)
+        self.updateWindow()
+
+    def updateWindow(self):
+        self.chosenColumns.sort()
+        self.unchosenColumns.sort()
+
+        self.unchoseColListbox.destroy()
+        self.choseColListbox.destroy()
+        self.addColButton.destroy()
+        self.removeColButton.destroy()
+
+        self.unchoseColVar = tk.StringVar(value=self.unchosenColumns)
+        self.choseColVar = tk.StringVar(value=self.chosenColumns)
+
+        self.unchoseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.unchoseColVar)
+        self.unchoseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+        self.choseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.choseColVar)
+        self.choseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+
+        self.addColButton = tk.Button(self, text="Add Column", command=self.addCol)
+        self.addColButton.pack()
+        self.removeColButton = tk.Button(self, text="Remove Column", command=self.removeCol)
+        self.removeColButton.pack()
```

Comment:
```suggestion
        self.unchoseColVar.set(self.unchosenColumns)
        self.choseColVar.set(self.chosenColumns)
```
Don't have to recreate the widgets, just need to update their list data.